package-lock.json

```diff
@@ -273,6 +273,11 @@
             "resolved": "https://registry.npmjs.org/@babel/helper-validator-identifier/-/helper-validator-identifier-7.10.4.tgz",
             "integrity": "sha512-3U9y+43hz7ZM+rzG24Qe2mufW5KhvFg/NhnNph+i9mgCtdTCtMJuI1TMkrIUiK7Ix4PYlRF9I5dhqaLYA/ADXw=="
         },
+        "@babel/helper-validator-option": {
```

Comment:
You have to commit the modifications linked to commons-ui and notistack only, see review of this PR https://github.com/gridsuite/gridmerge-app/pull/40